packages/contracts/test/dollar/BondingCurve.t.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../helpers/LiveTestHelper.sol";
+
+contract ZeroState is LiveTestHelper {
```

Comment:
why is it a liveTest ? can't it be a localTest ?

Reply:
I noticed most of the contracts make use of the live tests so i just followed the pattern. Any recommendations? @zgorizzo69

Reply:
Done. Will be available in next commit